postfix/datadog_checks/postfix/postfix.py

```diff
@@ -143,14 +143,17 @@ def _get_postqueue_stats(self, postfix_config_dir, tags):
 
         -- 1 Kbytes in 2 Requests.
         '''
-        for line in output.splitlines():
-            if '*' in line:
-                active_count += 1
-            elif '!' in line:
-                hold_count += 1
-            # Check the line starts with an ID
-            elif line[0:1].isalnum():
-                deferred_count += 1
+        lines = output.splitlines()
+        # Check output
+        if len(lines) > 1 and not output.startswith('Mail queue is empty'):
```

Comment:
what if len(lines) == 1 ?

Reply:
The first line is the header one, something like:
```
-Queue ID-  --Size-- ----Arrival Time---- -Sender/Recipient-------
DBDCF3206B5     313 Thu Mar 18 14:09:23  root@mycompany.com
...
```

So if len(lines) == 1 there is nothing to parse either